src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -8,7 +8,31 @@ public class RegistrationServiceImpl implements RegistrationService {
     private final StorageDao storageDao = new StorageDaoImpl();
 
     @Override
-    public User register(User user) {
-        return null;
+    public User register(User user) throws CustomException {
+        if (user.getAge() == null || user.getPassword() == null || user.getLogin() == null) {
+            throw new CustomException("User login,password or age is null!"
+                    + user.getAge() + user.getPassword() + user.getLogin());
```

Comment:
The exception message concatenates the user's age, password, and login directly, which may not be safe or informative. Consider providing a more user-friendly message without exposing sensitive information.